content/news/2020-11-13-release-plans-and-call-for-contrib.md

```diff
@@ -0,0 +1,48 @@
+
+## Why should I contribute to Haiku?
+
+- Haiku is open source, and licensed with a permissive license (MIT).
+- Haiku is inspired by BeOS, which is highly praised as an operating system; now, and back in the day.
+- Haiku offers a POSIX compatible operating system designed from scratch, with no legacy burden that other operating systems suffer from.
+- The Haiku experience does not suffer from fragmentation. All Haiku components are maintained in a single codebase, without having to rely on other teams for core system components. No flame wars between Z12 and Roadcountry supporters, or Leprechauns and Krusaders. We don't curse on the mailing lists as well.
+- Haiku is a GUI-first operating system, unlike most other FOSS systems. The Terminal is still a first-class citizen after booting the system, right there waiting for you.
+- Haiku is the only operating system that completely stays true to the [desktop metaphor](https://en.wikipedia.org/wiki/Desktop_metaphor). Tracker is the only remaining mainstream spatial desktop manager. If you are looking for an desktop experience close to the Classic Mac OS, Haiku is your guy.
+- Haiku offers a unique solution for package management, which most Linux distributions seem to suffer from. All Haiku packages are mountable disk images, therefore they do not litter your system with copied files everywhere. Updating and recovering the system is as easy as replacing a single file.
+- You might find it surprising that we respect the "everything-is-a-file" concept more than other \*nixes.
+
+If you are looking for more detailed content, Andrew Greimann has written an excellent [article](https://osvoyager.wordpress.com/2018/11/30/what-makes-beos-and-haiku-unique/) (2018) on the unique points of BeOS and Haiku. [Haiku User Guide](https://www.haiku-os.org/docs/userguide/en/contents.html) is an excellent source on the Haiku system itself and its components. probono has written an excellent series of Haiku articles (2019), which can be found [here](https://medium.com/@probonopd/my-first-day-with-haiku-shockingly-good-8930cad4bbb0) as well.
+
+## Other Means of Support
+
+If you are not a developer, but still want to support Haiku, we appreciate any support offer. Haiku [needs](https://www.haiku-os.org/community/getting-involved/) QA people, designers, documenters... you name it. Our website currently doesn't have a maintainer for instance.
+
+If you are able to support the project financially, please make a donation. We are looking for means to hire a full-time developer for an extended period of time, but we do not have the financial resources yet to accomplish this goal. Any [donation](https://www.haiku-inc.org/donate/) you make to Haiku, Inc. will help us realize this goal.
```

Comment:
We have enough money to pay someone for 1 to 2 years, so that's not true. (2 years is already an extended period, I think)

Reply:
1 year or 2 year with less-than-average salary without the necessary income to sustain it, is not a wise financial move. From what I've read, I believe Haiku, Inc. members agree on this.